cmd/ctr/commands/images/images.go

```diff
@@ -212,17 +212,21 @@ var checkCommand = cli.Command{
 			return err
 		}
 		defer cancel()
-		var (
-			contentStore = client.ContentStore()
-			tw           = tabwriter.NewWriter(os.Stdout, 1, 8, 1, ' ', 0)
-		)
-		fmt.Fprintln(tw, "REF\tTYPE\tDIGEST\tSTATUS\tSIZE\tUNPACKED\t")
+
+		var contentStore = client.ContentStore()
 
 		args := []string(context.Args())
 		imageList, err := client.ListImages(ctx, args...)
 		if err != nil {
 			return errors.Wrap(err, "failed listing images")
 		}
+		if len(imageList) == 0 {
+			log.G(ctx).Debugf("no images found")
+			return exitErr
```

Comment:
`exitErr` seems always `nil` here. Is this expected?

Reply:
yes.... Derek didn't think it good to return an error on no images, due I believe to the case where check is used to determine if any of the images in cache have an issue. I used exitErr by convention in the surrounding code in case someone ever sets it non nil at some future date.

Reply:
Thanks for the explanation. SGTM.